system-uptime-tracking/xml/art_system-uptime-tracking.xml

```diff
@@ -0,0 +1,375 @@
+      <dm:translation>yes</dm:translation>
+    </dm:docmanager>
+    <abstract>
+      <para>
+        The system uptime tracking feature monitors when a client system is up.
+        It works on an hourly basis&mdash;the system is considered up if it was
+        running at any part of the given hour. The information is stored in
+        &scc; (SCC) and Repository Mirroring Tool (RMT), and serves as a source
+        for analytic purposes.
+      </para>
+    </abstract>
+  </info>
+  <sect1 xml:id="uptime-requirements">
+    <title>Requirements</title>
+
+    <itemizedlist>
+      <listitem>
+        <para>
+          The version of the <package>suseconnect-ng</package> package has to be
+          1.10 or newer. Verify it by running <command>suseconnect
+          --version</command>.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          If the client system is behind RMT, the version of the
+          <package>rmt-server</package> package has to be 2.17 or newer. Verify
+          it by running <command>rpm -q rmt-server</command>.
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+  <sect1 xml:id="uptime-enable">
+    <title>Enabling the uptime tracking</title>
+
+    <para>
+      The uptime tracking is disabled by default. To enable it on the client
+      system, start its &systemd; timer and adjust the &suseconnect;
+      configuration.
+    </para>
+
+    <procedure>
+      <title>Enabling the &systemd; timer</title>
+      <step>
+        <para>
+          Install the <package>suse-uptime-tracker</package> package.
+        </para>
+<screen>&prompt.sudo;zypper install suse-uptime-tracker</screen>
+      </step>
+      <step>
+        <para>
+          Enable and start the
+          <systemitem class="daemon">suse-uptime-tracker</systemitem> service.
+        </para>
+<screen>&prompt.sudo;systemctl enable --now suse-uptime-tracker</screen>
+      </step>
+      <step>
+        <para>
+          Check if the service is running.
+        </para>
+<screen>&prompt.user;systemctl status suse-uptime-tracker</screen>
+      </step>
+    </procedure>
+
+    <procedure>
+      <title>Adjusting the &suseconnect; configuration</title>
+      <step performance="optional">
+        <para>
+          If you do not have a copy of the <filename>/etc/SUSEConnect</filename>
+          file, create it by copying the example file.
+        </para>
+<screen>&prompt.sudo;cp /etc/SUSEConnect.example /etc/SUSEConnect</screen>
+      </step>
+      <step>
+        <para>
+          Edit <filename>/etc/SUSEConnect</filename> and enable the uptime
+          tracking.
+        </para>
+<screen>enable_system_uptime_tracking: true</screen>
+      </step>
+      <step>
+        <para>
+          After enabling the uptime tracking, the timer checks if the system is
+          running every 15 minutes and stores the findings in the log files
```

Comment:
its a singular log file, I would s/files/file/ in this line and the one below, and s/These/this/ in the line below